File: sistema_mercado_java/src/modelo/ItemVenda.java
package modelo;

// import java.time.LocalDate;
// import java.time.Period;
// import java.time.format.DateTimeFormatter;
// import java.util.ArrayList;
// import java.util.List;

public class ItemVenda {

    private Produto produto;
    private int quantidade;

    public Produto getProduto() {
        return produto;
    }

    public int getQuantidade() {
        return quantidade;
    }

    public void setQuantidade(int quantidade) {
        this.quantidade = quantidade;
    }

    public void setProduto(Produto produto) {
        this.produto = produto;
    }

    public ItemVenda(Produto produto, int quantidade) {
        this.produto = produto;
        this.quantidade = quantidade;
    }
}

File: sistema_mercado_python/modelos/ItemVenda.py
class ItemVenda:
    def __init__(self, produto, quantidade):
        self._produto = produto
        self._quantidade = quantidade

    def get_produto(self):
        return self._produto

    def set_produto(self, produto):
        self._produto = produto

    def get_quantidade(self):
        return self._quantidade

    def set_quantidade(self, quantidade):
        self._quantidade = quantidade